milli/src/update/delete_documents.rs

```diff
@@ -9,6 +9,7 @@ use crate::facet::FacetType;
 use crate::{Index, BEU32, SmallString32, ExternalDocumentsIds};
 use crate::heed_codec::facet::{FieldDocIdFacetStringCodec, FieldDocIdFacetF64Codec, FieldDocIdFacetI64Codec};
 use super::ClearDocuments;
+use std::collections::hash_map::Entry;
```

Comment:
Could you please move this import above the first import (`anyhow`) and keep an empty line, in other words: put it at the top of the import and keep it separated with an empty newline, please?